src/components/DashboardHome/DashboardHome.jsx

```diff
@@ -13,124 +14,147 @@ import Sidebar from './Sidebar';
+            </div>
+          </div>
+          }
+        dismissible={false}
+        open
+      />
+    </div>
+  );
 
   render() {
+    const { hasProgramAccess } = this.state;
+    const { pageContext, isLoading } = this.props;
+    const { programUUID, programName } = pageContext;
+
     return (
       <IntlProvider locale="en">
         <Layout>
-          {
-           this.state.validAccess ? (
-             <>
-               <Helmet title="Master's Degree in Analytics" />
-               <main id="content">
-                 <Hero
-                   programTitle="Master's Degree in Analytics"
-                   organizationLogo={{
-            url: 'https://www.edx.org/sites/default/files/school/image/logo/gtx-logo-200x101.png',
-            alt: 'Georgia Tech Institute of Technology logo',
-          }}
-                   textureImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt-cyber-title_bg_img_440x400.jpg"
-                   coverImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt_cyber_campus_image_1000x400.jpg"
-                 />
-                 <div className="container py-5">
-                   <div className="row">
-                     <div className="col-xs-12 col-lg-7">
-                       <MainContent />
-                     </div>
-                     <MediaQuery minWidth={breakpoints.large.minWidth}>
-                       {matches => matches && (
-                       <aside className="col offset-lg-1">
-                         <Sidebar />
-                       </aside>
+          {isLoading ? (
+            <div className="d-flex justify-content-center align-items-center" style={{ height: 200 }}>
+              <div className="spinner-border text-primary" role="status">
+                <div className="sr-only">Loading program enrollments...</div>
+              </div>
+            </div>
+          ) : (
+            <>
+              {hasProgramAccess ? (
+                <>
+                  <Helmet title={programName} />
+                  <main id="content">
+                    <Hero
+                      programTitle={programName}
+                      organizationLogo={{
+                        url: 'https://www.edx.org/sites/default/files/school/image/logo/gtx-logo-200x101.png',
+                        alt: 'Georgia Tech Institute of Technology logo',
+                      }}
+                      textureImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt-cyber-title_bg_img_440x400.jpg"
+                      coverImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt_cyber_campus_image_1000x400.jpg"
+                    />
+                    <div className="container py-5">
+                      <div className="row">
+                        <div className="col-xs-12 col-lg-7">
+                          <MainContent programUUID={programUUID} />
```

Comment:
Passing the `programUUID` into `MainContent` as a prop (comes from pageContext) instead of having `MainContent` making a query for `programUUID` in a static query.